jujugui/static/gui/src/app/views/state.js

```diff
@@ -117,7 +117,7 @@ YUI.add('juju-app-state', function(Y) {
       @param {Object} state The current state object.
     */
     dispatch: function(state) {
-      var sections = ['app', 'sectionA', 'sectionB'];
+      var sections = ['app', 'sectionA', 'sectionB', 'sectionC'];
```

Comment:
yes I have a task to allow dynamic registering of sections...but this was a quick and dirty way to add this support for now